.github/workflows/ci.yml

```diff
@@ -69,8 +69,23 @@ jobs:
         uses: xom9ikk/dotenv@v2
         with:
           mode: test
+      - name: Install dependencies
+        # Not using test:unit dependency in Turbo because Datadog Test Visibility requires only the test suite to be run
```

Comment:
ps, i didn't understand this comment, can explain?

Reply:
ya nvm it still doesn't work, man-in-the-middle does not like prisma generate haha. need do outside. dd-agent can only trace the test itself, not the full turbo pipeline